app/src/main/java/com/andela/art/serialentry/SerialEntryActivity.java

```diff
@@ -21,7 +21,7 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_serial_entry);
 
-        Toolbar toolbar = findViewById(R.id.toolbarTB);
+        Toolbar toolbar = findViewById(R.id.toolbar);
```

Comment:
You could implement Data binding here, you can reach to @asce1062  to show you how they did it

Reply:
well noted.